backend/dataall/modules/maintenance/services/maintenance_service.py

```diff
@@ -0,0 +1,119 @@
+            return False
+
+    # Update the RDS table by changing mode to - ''
+    # Update the RDS table by changing the status to INACTIVE
+    # Enable all the ECS Scheduled task
+    @staticmethod
+    def stop_maintenance_window(engine):
+        logger.info('Stopping maintenance mode')
+        try:
+            with engine.scoped_session() as session:
+                maintenance_record = MaintenanceRepository(session).get_maintenance_record()
+                if maintenance_record.status == MaintenanceStatus.INACTIVE.value:
+                    logger.error('Maintenance window already in INACTIVE state. Cannot stop maintenance window')
+                    return False
+                MaintenanceRepository(session).save_maintenance_status_and_mode(
+                    maintenance_status='INACTIVE', maintenance_mode=''
+                )
+            # Enable scheduled ECS tasks
+            ecs_scheduled_rules = ParameterStoreManager.get_parameters_by_path(
+                region=os.getenv('AWS_REGION', 'eu-west-1'),
+                parameter_path=f"/dataall/{os.getenv('envname', 'local')}/ecs/ecs_scheduled_tasks/rule",
+            )
+            logger.debug(ecs_scheduled_rules)
+            ecs_scheduled_rules_list = [item['Value'] for item in ecs_scheduled_rules]
+            event_bridge_session = EventBridge(region=os.getenv('AWS_REGION', 'eu-west-1'))
+            event_bridge_session.enable_scheduled_ecs_tasks(ecs_scheduled_rules_list)
+            return True
+        except Exception as e:
+            logger.error(f'Error occurred while stopping maintenance window due to {e}')
+            return False
+
+    # Checks if all ECS tasks in the data.all infra account have completed
+    # Updates the maintenance status and returns maintenance record
+    @staticmethod
+    def get_maintenance_window_status(engine):
+        logger.info('Checking maintenance window status')
+        try:
+            with engine.scoped_session() as session:
+                maintenance_record = MaintenanceRepository(session).get_maintenance_record()
+                if maintenance_record.status == MaintenanceStatus.PENDING.value:
+                    # Check if ECS tasks are running
+                    ecs_cluster_name = ParameterStoreManager.get_parameter_value(
+                        region=os.getenv('AWS_REGION', 'eu-west-1'),
+                        parameter_path=f"/dataall/{os.getenv('envname', 'local')}/ecs/cluster/name",
+                    )
+                    if Ecs.is_task_running(cluster_name=ecs_cluster_name):
+                        return maintenance_record
+                    else:
+                        maintenance_record.status = MaintenanceStatus.ACTIVE.value
+                        session.commit()
+                        return maintenance_record
+                else:
+                    logger.info('Maintenance window is not in PENDING state')
```

Comment:
nit: can we show a better log for this? Maybe show the actual status

Reply:
Sure . Will do!